### PR TITLE
Third weekly report

### PR DESCRIPTION
**Pull request by robin-debry**

### CODE REVIEW

File: Documents/Management/WeeklyReports/weeklyReportThirdWeek.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Manage each component with LEDs.
+2. Finish writing technical specifications.
+3. Continue reverse engineering existing software with LEDs.
+4. Attempt to implement project functionality without hardware but with LEDs.
+
+## Achievements
+
+1. Manage each component with LEDs
+   - during this week, the team managed to simulate the functionality of each component with LEDs.
```
Comment (MathiasGagnepain) [inline Documents/Management/WeeklyReports/weeklyReportThirdWeek.md:19]: If you use Uppercase do it for each sentence

File: Documents/Management/WeeklyReports/weeklyReportThirdWeek.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+The difficulties encountered during this week were mainly due to the hardware components. The team had trouble with the hardware components, as the components did not work as expected. The team tried to solve the problems without success and decided to use LEDs to simulate the functionality of the project.
+
+## Next week's plan
+
+1. Documents
+   - Continue the test plan.
+   - Continue the user manual.
+
+2. Hardware components
+   - The team will have a new battery to test the project's functionality.
```
Comment (HiNett) [inline Documents/Management/WeeklyReports/weeklyReportThirdWeek.md:44]: Could you please explain a little bit the part with the battery. Where does the battery came from ?

**Review by HiNett — CHANGES_REQUESTED** [review]

**Review by MathiasGagnepain — CHANGES_REQUESTED** [review]
An uppercase is missing.

File: Documents/Management/WeeklyReports/weeklyReportThirdWeek.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ The difficulties encountered during this week were mainly due to the hardware co
    - Continue the user manual.
 
 2. Hardware components
-   - The team will have a new battery to test the project's functionality.
+   - The team will have a new battery to test the project's functionality, This new battery was ordered last week by Mathias because we thought the battery was necessary to test the project.
```
Comment (MathiasGagnepain) [inline Documents/Management/WeeklyReports/weeklyReportThirdWeek.md:44]: After a comma we don't have to use a uppercase.

**Review by MathiasGagnepain — CHANGES_REQUESTED** [review]
You have to make few changes.

File: Documents/Management/WeeklyReports/weeklyReportThirdWeek.md (hunk 3 above)
Comment (HiNett) [reply Documents/Management/WeeklyReports/weeklyReportThirdWeek.md:44]: True.

**Review by MathiasGagnepain — APPROVED** [review]
I thought you were going to remove the capital letter but it ok with a dot.

**Review by HiNett — APPROVED** [review]
I'm proud of you, Robin. You've shown us once again your ability to solve problems in a short space of time. Keep up the good work, you're a professional.
557de94d-cfd5-49f1-aed5-dad4b10022c0_text

**Pull request merged**